timed/projects/filters.py

```diff
@@ -44,7 +43,7 @@ def filter_has_manager(self, queryset, name, value):
             )
         )
 
-    def filter_has_reviewer(self, queryset, name, value):
+    def filter_has_reviewer(self, queryset, name, value):  # pragma: no cover
```

Comment:
Please write tests instead of using `# pragma: no cover`. Especially for parts that contain non-trivial code. Same for the filter method above.